crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -442,6 +442,21 @@ pub fn derive_system_label(input: TokenStream) -> TokenStream {
     derive_label(input, &trait_path)
 }
 
+#[proc_macro_attribute]
+pub fn system_label(_attr: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
This probably needs some documentation. It should also indicate which traits are automatically derived.

Reply:
If there is a desire to merge this, I will add docs.